app/views/inventories/index.html.erb

```diff
@@ -0,0 +1,22 @@
+
+<div id="inventory-list">
+  <% if current_user %>
+    <h1>Your Inventories</h1>
+    <div class="d-flex justify-content-end">
+      <%= link_to "add inventory", new_inventory_path, class: "btn btn-primary"%>
+    </div>
+    <% if @inventories.empty? %>
+      <p>No inventories yet.</p>
+    <% else %>
+      <% @inventories.each do |inventory| %>
+        <div class="inventory-card">
+          <div class="inventory-card-header gap-5">
+              <%= link_to inventory.name, inventory_path(inventory.id), class: "inventory-card-name" %>
+              <p class="inventory-card-description text-start"><%= inventory.description %></p> 
+          </div>
+          <%= button_to "Remove", inventory_path(inventory.id), method: :delete, data: { confirm: "Are you sure?", turbo_confirm: "Are you sure?" }, class: "inventory-card-link inventory-card-link-remove text-danger" %>
```

Comment:
- [ ] 🔐 According to the project guidelines, only the rightful owner should have the authority to delete a inventory item. It's crucial to enhance security by verifying that the user attempting the deletion is indeed the rightful owner. To achieve this, consider implementing an authorization mechanism, such as CancanCan or any other suitable solution. This mechanism will add an additional layer of protection by ensuring that only users with the appropriate permissions can access the delete functionality. This way, we strengthen the security of our application and guarantee that only authorized individuals can modify or remove inventory items. 🚀